tests/bson/bug0974-001.phpt

```diff
@@ -44,7 +44,7 @@ object(stdClass)#%d (%d) {
   }
 }
 object(stdClass)#3 (2) {
-  [%r(0|"0")%r]=>
+  ["0"]=>
```

Comment:
There are more occurrences above and below to replace in this file. Best to search for `")%r]=>` throughout the test files to catch all of these.